ros-dev-tools/README.md

```diff
@@ -0,0 +1,11 @@
+# ROS Dev Tools
+
+`ros-dev-tools` contains packages that are of general use to ROS developers but which are not dependencies of any particular packages within a ROS distribution.
+
+### Packages included in this variant
+
+* ros-build-essential
```

Comment:
Nit: I think you can link this to the sibling package with something like:

```suggestion
* [ros-build-essential](../ros-build-essential/README.md)
```

Reply:
I don't want to do this because these READMEs will be installed installed on host systems where these may not be direct relatives.